changelogs/.plugin-cache.yaml

```diff
@@ -0,0 +1,170 @@
+objects:
+  role: {}
+plugins:
+  become: {}
+  cache: {}
+  callback: {}
+  cliconf:
+    sonic:
+      description: Use sonic cliconf to run command on Dell OS10 platform
+      name: sonic
+      version_added: null
+  connection: {}
+  httpapi:
+    sonic:
+      description: HttpApi Plugin for devices supporting Restconf SONIC API
+      name: sonic
+      version_added: 1.0.0
+  inventory: {}
+  lookup: {}
+  module:
+    sonic_aaa:
+      description: Manage AAA and its parameters
+      name: sonic_aaa
+      namespace: ''
+      version_added: 1.1.0
+    sonic_api:
+      description: Manages REST operations on devices running Enterprise SONiC
+      name: sonic_api
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp:
+      description: Manage global BGP and its parameters
+      name: sonic_bgp
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_af:
+      description: Manage global BGP address-family and its parameters
+      name: sonic_bgp_af
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_as_paths:
+      description: Manage BGP autonomous system path (or as-path-list) and its parameters
+      name: sonic_bgp_as_paths
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_communities:
+      description: Manage BGP community and its parameters
+      name: sonic_bgp_communities
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_ext_communities:
+      description: Manage BGP extended community-list and its parameters
+      name: sonic_bgp_ext_communities
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_neighbors:
+      description: Manage a BGP neighbor and its parameters
+      name: sonic_bgp_neighbors
+      namespace: ''
+      version_added: 1.0.0
+    sonic_bgp_neighbors_af:
+      description: Manage the BGP neighbor address-family and its parameters
+      name: sonic_bgp_neighbors_af
+      namespace: ''
+      version_added: 1.0.0
+    sonic_command:
+      description: Runs commands on devices running Enterprise SONiC
+      name: sonic_command
+      namespace: ''
+      version_added: 1.0.0
+    sonic_config:
+      description: Manages configuration sections on devices running Enterprise SONiC
```

Comment:
Is this like a CLI config command?

Reply:
That's exactly what it is. (Again, this is inherited content that was previously published, but I can try editing it to see if that fixes the changelogs that customers will see.)

Reply:
Actually, it enables running of exec commands, also (not just config. commands).

Reply:
never mind, this is auto-generated code. We should clarify this in the future.